src/platform/Linux/dbus/bluez/DbusBluez.xml

```diff
@@ -187,9 +187,11 @@
     <property name="Includes" type="as" access="read"/>
     <property name="LocalName" type="s" access="read"/>
     <property name="Appearance" type="q" access="read"/>
-    <property name="Duration" type="q" access="read"/>
-    <property name="Timeout" type="q" access="read"/>
+    <!-- <property name="Duration" type="q" access="read"/> -->
+    <!-- <property name="Timeout" type="q" access="read"/> -->
     <!-- <property name="SecondaryChannel" type="s" access="read"/> -->
+    <property name="MinInterval" type="u" access="read"/>
```

Comment:
for this experimental feature, what is the minimum Bluez version? I am worried about the compatibility issue, for example, If I put the code in raspi where it has default bluez, does it work?

Reply:
It required BlueZ >= 5.56. However, for versions lower than that it was impossible to set advertising intervals, so I would say that BlueZ < 5.56 is not compatible with Matter. Also, since these properties are part of experimental features, one has to add `-E`/`--experimental` to `bluetoothd` command line, e.g.:

```shell
sudo mkdir -p /etc/systemd/system/bluetooth.service.d
cat << EOF | sudo tee /etc/systemd/system/bluetooth.service.d/override.conf
[Service]
ExecStart=
ExecStart=<ABS_PATH_TO_bluetoothd> --experimental
EOF
```

Reply:
I've added an instruction how to enable experimental features in BlueZ to the RPi section in BUIDING.md.